js/taskjobs.js

```diff
@@ -434,11 +434,17 @@ function agents_chart(chart_id) {
                         "</th></tr>"
                      );
                       $.each(run.logs, function(log_index, log) {
+                        var logComment = log['log.comment'];
+                        var parts = logComment.split(/(<a.*?>|<\/a>)/);
+                        if (parts.length > 1) {
+                            logComment = __(parts[0], 'glpiinventory') + parts[1] + parts[2] + parts[3] + parts[4];
+                        }
```

Comment:
Would something like this works? The idea is it still works work even if we have more parts than expected.
```suggestion
                        var logComment = log['log.comment'];
                        var parts = logComment.split(/(<a.*?>|<\/a>)/);
                        
                        var logComment = __(parts[0], 'glpiinventory'); 
                        var logComment += parts.slice(1).join(' ');
```

Reply:
Note: with original proposition, log won't be translated if there are no link (`parts.lenght == 1` is not handled)